demo/src/app/app.config.ts

```diff
@@ -11,7 +11,7 @@ export class AppConfig extends AbstractAppConfig {
     'document_management_url': '/documents',
     'login_url': '/login',
     'oauth2_client_id': 'ccd_gateway',
-    'postcode_lookup_url': '/addresses?postcode=${postcode}',
+    'postcode_lookup_url': 'https://api.ordnancesurvey.co.uk/places/v1/addresses/postcode?postcode=${postcode}&key=${key}',
```

Comment:
is this going through the CCD Gateway yes?

Reply:
Yes this is demo app which I had configured to run locally without the gateway. May be a good idea it to go through gateway as well, I will fix it.